Documentation/compatibility/wpf-selector-selectionchanged-and-selectedvalue.md

```diff
@@ -0,0 +1,56 @@
+## Selector SelectionChanged event and SelectedValue property
+
+### Scope
+Minor
+
+### Version Introduced
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+When its selection changes, a `T:System.Windows.Controls.Primitives.Selector` now updates the value of its
```

Comment:
It would be best to include the version here. So, "Starting with the .NET Framework 4.7.1, when..." In that case, "now" can be removed.